src/hooks/useAccount.ts

```diff
@@ -0,0 +1,24 @@
+import { useBearStore } from '@/store/store';
+import useNavigate from './useNavigate';
+import { useEffect } from 'react';
+
+export default function useAccount() {
+  const { publicKey, resetAccount } = useBearStore(
+    ({ resetAccount, account }) => ({
+      publicKey: account.publicKey,
+      resetAccount,
+    }),
+  );
+  const { handleNavigate } = useNavigate();
+  const checkedExistPublicKeyInStore = () => {
+    if (publicKey.length !== 56) {
```

Comment:
What's 56? What does mean? Let's save that magic number inside a variable with a descriptive name, please.